translations/en.json

```diff
@@ -6,8 +6,8 @@
     "more": "more"
   },
   "homepage": {
-    "siteTitle": "OpenTechSchool",
-    "siteDescription": "OpenTechSchool is a movement aiming to offer free tech education.",
+    "siteTitle": "Free tech education",
```

Comment:
i would still keep "OpenTechSchool" here. 
Like: "OpenTechSchool - Free tech education" os similar

Reply:
It's not in Meta.tsx: https://github.com/OpenTechSchool/websitenext/pull/125/files#diff-a81429a07d624e4d5625bcfcd0d65011R13

Reply:
ah! I misread the brackets in the ternary